locksmith/src/config/config.ts

```diff
@@ -77,6 +77,8 @@ const config = {
     exportsBucket: 'uploads',
   },
   recaptchaSecret: process.env.RECAPTCHA_SECRET,
+  gitcoinApiKey: process.env.GITCOIN_API_KEY,
```

Comment:
Can you please point to where we can find these please?
Ideally you would add this as a comment in this file (just above these 2 lines). Something like:

>  The gitcoin API keyis required to make the gitcoin captcha work!

Reply:
sure thing. i already did this in the PR descriotion. provided a link to the setup instructions in there. would further  add it as a comment